cloud-haskell.cabal

```diff
@@ -47,10 +47,10 @@ Flag p2p
   Manual:  True
 
 Library
-  Build-depends:     
+  Build-depends:
                      rank1dynamic >= 0.2.0.0 && < 0.3,
                      distributed-static >= 0.3.0.0 && < 0.4,
-                     network-transport >= 0.4.1.0 && < 0.5,
+                     network-transport >= 0.4.1.0 && < 0.5
```

Comment:
Have you really checked that this still compiles? It seems that the only change on this line introduces a syntax error.